deploy/charts/cert-manager/templates/cainjector-deployment.yaml

```diff
@@ -71,30 +71,32 @@ spec:
           - --leader-election-retry-period={{ .retryPeriod }}
           {{- end }}
           {{- end }}
-          {{- if .Values.cainjector.extraArgs }}
-{{ toYaml .Values.cainjector.extraArgs | indent 10 }}
+          {{- with .Values.cainjector.extraArgs }}
+          {{- toYaml . | nindent 10 }}
           {{- end }}
           env:
           - name: POD_NAMESPACE
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          {{- if .Values.cainjector.containerSecurityContext }}
+          {{- with .Values.cainjector.containerSecurityContext }}
           securityContext:
-            {{- toYaml .Values.cainjector.containerSecurityContext | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- with .Values.cainjector.resources }}
```

Comment:
Here, an additional with statement was added, this allows us to not specify a value for resources in the values.yaml file an still get a valid render.

Reply:
Confirmed that this change results in the "resources" block for this Deployment not being rendered in this PR. 

On the `master` branch, the deployment has the following block:

```text
           resources:
             {}
```